src/cli_config.rs

```diff
@@ -123,9 +123,8 @@ pub struct DecodeArgs {
     /// The algorithm used to sign the JWT
     #[clap(long = "alg", short = 'A')]
     #[clap(value_enum)]
-    #[clap(default_value = "HS256")]
     #[clap(value_parser)]
-    pub algorithm: SupportedAlgorithms,
+    pub algorithm: Option<SupportedAlgorithms>,
```

Comment:
I'm not sure what the purpose of this change is. Can you elaborate on why this is needed?

Reply:
The algorithm used for the JWT can be derived from the JWT header so making this optional would be better UX IMO